en/components/grid.md

```diff
@@ -688,6 +831,8 @@ Inputs available on the **IgxGridColumnComponent** to define columns:
 |`sortable`|boolean|Set column to be sorted or not|
 |`editable`|boolean|Set column values to be editable|
 |`filterable`|boolean|Set column values to be filterable|
+|`hasSummary`| boolean  |Set the specific column to have a summaries or not|
```

Comment:
Sets whether or not the specific column has summaries enabled.
Assuming that's what it does :)